.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+name: build-and-test
+on:
+  push:
+    branches:
+      - master
+      - iximiuz/github-actions
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out docker-slim/docker-slim repo
+        uses: actions/checkout@v3
+
+      - name: Set up Go development environment
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.15.0'
```

Comment:
Would it make sense to try and build docker slim for all supported go versions?

Reply:
To be honest, I don't know. Until this MR, `go.mod` had `go 1.13`, but I couldn't actually build the project with anything older than `1.15`. So, I bumped the version up. However, I'm not aware of the inverse scenarios when something wouldn't build with a newer Go version than specified in the `go.mod`. The language is kind of famous for its backward compatibility guarantees. But I'm not 100% sure 😄

Reply:
Good point about the `latest`!